spec/datadog/tracing/configuration/settings_spec.rb

```diff
@@ -0,0 +1,656 @@
+# typed: false
+
+require 'spec_helper'
+
+require 'securerandom'
+
+require 'datadog/core/configuration/settings'
+require 'datadog/tracing/configuration/ext'
+require 'datadog/tracing/flush'
+require 'datadog/tracing/sampling/priority_sampler'
+require 'datadog/tracing/tracer'
+require 'datadog/tracing/writer'
+
+RSpec.describe Datadog::Tracing::Configuration::Settings do
+  # TODO: Core::Configuration::Settings directly extends Tracing::Configuration::Settings
+  #       In the future, have tracing add this behavior itself. For now,
+  #       just use the core metrics class to drive the tests.
+  subject(:settings) { Datadog::Core::Configuration::Settings.new(options) }
+
+  let(:options) { {} }
+
+  describe '#tracing' do
```

Comment:
These specs are now duplicated: one copy here, another in `spec/datadog/core/configuration/settings_spec.rb`.
They are pretty long hard to keep in sync.
I was recently making changes to `propagation_extract_style` & `propagation_inject_style` and would have to make duplicate version of the same test in both places after this PR.

Is the goal to remove these specs from `spec/datadog/core/configuration/settings_spec.rb` after the refactoring is done? If, should we do this now, as to avoid any possible divergence between this file and `spec/datadog/core/configuration/settings_spec.rb`?

Reply:
I messed up. Meant to delete them from `core/configuration/settings_spec.rb`. Should be fixed now.